core/src/main/java/org/apache/iceberg/rest/RESTSessionCatalog.java

```diff
@@ -116,6 +116,7 @@ public class RESTSessionCatalog extends BaseViewSessionCatalog
   private static final String DEFAULT_FILE_IO_IMPL = "org.apache.iceberg.io.ResolvingFileIO";
   private static final String REST_METRICS_REPORTING_ENABLED = "rest-metrics-reporting-enabled";
   private static final String REST_SNAPSHOT_LOADING_MODE = "snapshot-loading-mode";
+  static final String REST_NAMESPACE_SEPARATOR = "rest-namespace-separator";
```

Comment:
Probably worth double checking if there's a good place to document this? I don't think we have a dedicated client side catalog configuration document  for REST , maybe we can take that as a separate issue. But if there's anywhere else to add this in the interim, that would be good.

Reply:
yeah I agree that we currently don't have a good place to document this but I'll check whether we can add this somewhere

Reply:
I've mentioned this in the OpenAPI spec